EIPS/eip-6666.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 6666
```

Comment:
```suggestion
eip: 6953
```
Assigning PR number as EIP number.  PR #6666 is unrelated.

Reply:
I tried :-(